app/script/components/message.js

```diff
@@ -66,6 +66,9 @@ class Message {
     this.actionsViewModel = actionsViewModel;
 
     this.showLikes = ko.observable(false);
+    this.hasReadReceiptsTurnedOn =
+      this.conversationRepository.active_conversation() &&
```

Comment:
Let's use the `conversation` parameter that is injected in the constructor instead of relying on the current active convo (which might not be the one linked to this message at some point in time)

Reply:
```suggestion
this.hasReadReceiptsTurnedOn = conversation.expectsReadConfirmation()
```